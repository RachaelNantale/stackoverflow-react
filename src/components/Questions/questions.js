import React from 'react';
import PropTypes from 'prop-types';
import './questions.scss';

const Questions = ({ question }) => (
  <div className="container">
    <div className="text-section">
      <div>
        <h4>
          {question.title}
        </h4>
        <hr />
        <div>
          {question.Time}
        </div>
        <div>
          {question.By}
        </div>
      </div>
    </div>
    <hr />
  </div>
);
Questions.propTypes = {
  question: PropTypes.shape({}),
};
Questions.defaultProps = {
  question: [],
};
export default Questions;
