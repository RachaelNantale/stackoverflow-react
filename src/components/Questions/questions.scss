.text-section {
  border: 1px solid lightgrey;
  border-radius: 5px;
  padding: 10px;
  margin:15px;
  background-color: rgba(12,13,14,0.05);
}