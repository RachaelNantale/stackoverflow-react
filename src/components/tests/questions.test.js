import React from 'react';
import { shallow } from 'enzyme';
import Questions from '../Questions/questions';

it('renders the Questions component', () => {
  const QuestionComponent = shallow(<Questions />);
  expect(QuestionComponent).toMatchSnapshot();
});
