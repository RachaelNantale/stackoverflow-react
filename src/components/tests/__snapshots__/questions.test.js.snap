// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Questions component 1`] = `
<div
  className="container"
>
  <div
    className="text-section"
  >
    <div>
      <h4 />
      <hr />
      <div />
      <div />
    </div>
  </div>
  <hr />
</div>
`;
